NeoMathEngine/src/CPU/CpuMathEngineDnnDropout.cpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright © 2017-2020 ABBYY Production LLC
+/* Copyright © 2017-2024 ABBYY Production LLC
```

Comment:
Super-minor comment: new header is "_Copyright © 2017-2024 ABBYY_", " _Production LLC_" should be deleted